internal/schemas.ts

```diff
@@ -810,15 +810,11 @@ const CompressedVideo: FoxgloveMessageSchema = {
     {
       name: "data",
       type: { type: "primitive", name: "bytes" },
-      description:
-        "Compressed video frame data.\n\nFor packet-based video codecs this data must begin and end on packet boundaries (no partial packets), and must contain enough video packets to decode exactly one image (either a keyframe or delta frame). Note: Foxglove does not support video streams that include B frames because they require lookahead.",
-    },
-    {
-      name: "format",
-      type: { type: "primitive", name: "string" },
-      description: `Video format.
+      description: `Compressed video frame data.
```

Comment:
Should we point here to the `format` field for more info on what this should look like?

Reply:
I figured since they are right next to each other maybe it’s fine? Could add it for redundancy though.